src/components/Warnings/WarningsModal.jsx

```diff
@@ -1,106 +1,106 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable spaced-comment */
-/* eslint-disable no-alert */
-/* eslint-disable no-undef */
-/* eslint-disable react/no-unescaped-entities */
-/* eslint-disable react/destructuring-assignment */
-import { Modal, ModalHeader, ModalBody, ModalFooter, Button, Row } from 'reactstrap';
+// /* eslint-disable no-unused-vars */
```

Comment:
Instead of commenting out all the code in the file, it would be better to delete the file.